src/main/java/core/service/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.service;
+
+public interface OperationHandler {
+    int getSign();
```

Comment:
you shouldn't return here sign, you should have such method handle(FruitTransaction fruitTransaction), where you somehow update your storage